docs/hsi.xml

```diff
@@ -0,0 +1,875 @@
+<?xml version="1.0"?>
+<!DOCTYPE book PUBLIC "-//OASIS//DTD DocBook XML V4.3//EN"
+               "http://www.oasis-open.org/docbook/xml/4.3/docbookx.dtd">
+<reference id="hsi">
+  <title>Host Security ID Specification</title>
+  <warning>
+    <para>
+      This specification is still in active development: it is incomplete,
+      subject to change, and may have errors; use this at your own risk.
+      It is based on publicly available information.
+    </para>
+  </warning>
+
+<!--
+  <info>
+    <authorgroup>
+      <author>
+        <personname>
+          <firstname>Richard</firstname>
+          <surname>Hughes</surname>
+        </personname>
+        <affiliation>
+          <orgname>Red Hat</orgname>
+        </affiliation>
+      </author>
+      <author>
+        <personname>
+          <firstname>Mario</firstname>
+          <surname>Limonciello</surname>
+        </personname>
+        <affiliation>
+          <orgname>Dell</orgname>
+        </affiliation>
+      </author>
+      <author>
+        <personname>
+          <firstname>Alex</firstname>
+          <surname>Bazhaniuk</surname>
+        </personname>
+        <affiliation>
+          <orgname>Eclypsium</orgname>
+        </affiliation>
+      </author>
+      <author>
+        <personname>
+          <firstname>Alex</firstname>
+          <surname>Matrosov</surname>
+        </personname>
+        <affiliation>
+          <orgname>Nvidia</orgname>
+        </affiliation>
+      </author>
+    </authorgroup>
+  </info>
+-->
```

Comment:
If these weren't commented out, how do they actually render?  Or do they?

Reply:
I commented them out for two reasons:

 * I'd like people to sign-off the final version, possibly when the ARM/AMD stuff has landed too
 * The rendering of the names+vendors is simply rubbish in the gtk-doc style sheet -- if we **do** want the author block then I might resort to `<para>` and `<itemizedlist>` as a hack.

![Screenshot_2020-07-28 Host Security ID Specification fwupd Reference Manual](https://user-images.githubusercontent.com/151380/88639104-4fd0dc00-d0b4-11ea-88c9-dec461c0078a.png)

Reply:
Ah OK, thanks.